src/State.ts

```diff
@@ -98,13 +104,26 @@ export function versionControl() {
       for (const t of newStatues) state.value.chars[key].statues[t] = 0;
     }
   }
+  if (state.value.version < "0.2.4") {
+    let colors = ["Orange", "Green", "Purple", "Yellow"];
```

Comment:
By typing `colors: Color[]`, the following type errors inside the for loop will be resolved

Reply:
This has been applied